nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/test/DirectoryEntriesTest.java

```diff
@@ -105,6 +113,23 @@ public void testLocalDirectoryEntries() throws Exception {
         JSONAssert.assertEquals(continentLocalContentJson, result.getString(), JSONCompareMode.NON_EXTENSIBLE);
     }
 
+    @Test
+    public void shouldFailWhenDuplicatesEntriesExist() {
+        String directoryName = "anyDirectory";
+        SQLDirectory directory = (SQLDirectory) directoryService.getDirectory(directoryName);
+        Schema schema = schemaManager.getSchema(directory.getSchema());
+        Session session = directory.getSession();
+
+        try {
+            DirectoryCSVLoader.loadData("testdirectorydata/continent_with_duplicates_entries.csv",
+                    DEFAULT_DATA_FILE_CHARACTER_SEPARATOR, schema, session::createEntry);
+        } catch (DirectoryException de) {
+            assertTrue(de.getMessage().contains("middleearth"));
```

Comment:
As there are already `assertEquals` on the exception message on other tests, I think we can do the same here.